backend/include/llvm/IR/IntrinsicsRISCVBuddyExt.td

```diff
@@ -20,6 +20,12 @@
 let TargetPrefix = "riscv" in
 def int_riscv_mvin : Intrinsic<[],[llvm_i64_ty, llvm_i64_ty],[]>;
 
+let TargetPrefix = "riscv" in
+def int_riscv_mvin2 : Intrinsic<[],[llvm_i64_ty, llvm_i64_ty],[]>;
+
+let TargetPrefix = "riscv" in
+def int_riscv_mvin3 : Intrinsic<[],[llvm_i64_ty, llvm_i64_ty],[]>;
+
```

Comment:
Intrinsic<[], [llvm_i64_ty, llvm_i64_ty], []>;

Reply:
I don't see the difference.